### ISSUE TITLE
[Bug]: Multi-kick only partially functional

### ISSUE THREAD

**Issue by twstagg** [Yimura/YimMenu#1978]
### Describe the bug

As of [a47af68](https://github.com/YimMenu/YimMenu/commit/a47af68384040e6330b863dcf837026eaf4afcf4) this function no longer works.

### Steps To Reproduce

1. Join public session
2. See a cheater try to do something to you
3. Decide to kick them using Multi-kick button
4. Use Multi-kick button to kick
5. Observe the person leave game, but immediately following the message of them leaving, the game freezes up and you have to force quit it.

### Expected Behavior

Ideally, the game would not freeze when kicking a malicious player

### Logs and/or screenshots

```
[21:36:59.4975009][DEBUG/logger.cpp:28] Using simple logger.
[21:37:02.3380434][INFO/main.cpp:65] Yim's Menu Initializing
[21:37:02.3380708][INFO/main.cpp:66] Git Info
	Branch:	master
	Hash:	a47af68384040e6330b863dcf837026eaf4afcf4
	Date:	Sat Aug 12 20:45:28 2023
[21:37:02.3380816][DEBUG/thread_pool.cpp:21] Resizing thread pool from 0 to 4
[21:37:02.3388533][INFO/main.cpp:69] Thread pool initialized.
[21:37:02.3413357][INFO/main.cpp:72] Settings Loaded.
[21:37:02.3414037][DEBUG/thread_pool.cpp:95] Thread 1784 executing settings.cpp:18
[21:37:02.3419519][INFO/pointers.hpp:94] GTA5 pointers cache is up to date, using it.
[21:37:02.3419584][INFO/pointers.cpp:1548] Loading pointers instance from cache
[21:37:02.3419599][INFO/pointers.cpp:1557] Pointers cache: Loading 181 fields from the cache
[21:37:02.3433765][DEBUG/module.cpp:50] Waiting for socialclub.dll...
[21:37:04.0332192][INFO/pointers.hpp:94] Social Club pointers cache is up to date, using it.
[21:37:04.0332355][INFO/pointers.cpp:1548] Loading pointers instance from cache
[21:37:04.0332383][INFO/pointers.cpp:1557] Pointers cache: Loading 5 fields from the cache
[21:37:04.0332599][INFO/main.cpp:75] Pointers initialized.
[21:37:04.0334688][INFO/main.cpp:78] Byte Patch Manager initialized.
[21:37:04.6449836][INFO/main.cpp:81] Renderer initialized.
[21:37:04.9124413][INFO/main.cpp:85] Fiber pool initialized.
[21:37:05.0363512][INFO/main.cpp:88] Translation Service initialized.
[21:37:05.0366669][INFO/main.cpp:91] Hooking initialized.
[21:37:05.0400598][DEBUG/gta_data_service.cpp:162] Loading data from cache.
[21:37:05.0400660][INFO/gta_data_service.cpp:174] Loading 1029 peds from cache.
[21:37:05.0402681][INFO/gta_data_service.cpp:192] Loading 815 vehicles from cache.
[21:37:05.0404435][INFO/gta_data_service.cpp:209] Loading 117 weapons from cache.
[21:37:05.0404449][INFO/gta_data_service.cpp:210] Loading 378 weapon components from cache.
[21:37:05.0404539][DEBUG/gta_data_service.cpp:168] Loaded all data from cache.
[21:37:05.0441985][INFO/tunables_service.cpp:21] Loading tunables from cache
[21:37:05.0442245][INFO/main.cpp:111] Registered service instances...
[21:37:05.0442969][DEBUG/thread_pool.cpp:95] Thread 1788 executing tunables_service.cpp:24
[21:37:05.0444323][INFO/main.cpp:132] Scripts registered.
[21:37:05.0625130][INFO/main.cpp:135] Hooking enabled.
[21:37:05.0625473][INFO/main.cpp:138] Dynamic native hooker initialized.
[21:37:05.0680298][INFO/lua_module.cpp:226] Loaded CasinoPacino.lua
[21:37:05.0718536][INFO/lua_module.cpp:226] Loaded YimNightClub_[unknowncheats.me]_.lua
[21:37:05.0719196][INFO/main.cpp:141] Lua manager initialized.
[21:37:05.0720058][DEBUG/thread_pool.cpp:95] Thread 1776 executing player_database_service.cpp:276
[21:37:49.9612980][INFO/backend.cpp:51] Starting script: Self
[21:37:49.9613241][INFO/backend.cpp:76] Starting script: Weapons
[21:37:49.9613358][INFO/backend.cpp:99] Starting script: Vehicles
[21:37:49.9613525][INFO/backend.cpp:115] Starting script: Miscellaneous
[21:37:49.9613577][INFO/backend.cpp:135] Starting script: Remote
[21:37:49.9613614][INFO/backend.cpp:150] Starting script: Ls customs
[21:37:49.9613640][INFO/backend.cpp:162] Starting script: Rainbow paint
[21:37:49.9613675][INFO/backend.cpp:174] Starting script: Disable Control Action
[21:37:49.9613718][INFO/backend.cpp:187] Starting script: World
[21:39:04.1455815][WARN/reaction.cpp:25] Received Send To Interior from Porknuts435 (233411927)
[21:40:11.5577666][WARN/reaction.cpp:25] Received Send To Interior from VladimireLenin (97715954)
[21:55:35.6759308][WARN/notify.hpp:30] Blocked rope crash from theo061997 (231858834)
[21:55:35.6759628][WARN/reaction.cpp:25] Received Crash from theo061997 (231858834)
```

### Store

Steam

### Additional context

In this circumstance, i kicked theo061997 and he was kicked. However, my game froze and I had to restart GTA 5 and join a new session afterward.

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by Yimura** [comment]
Duplicate of #1973

Or is it?

**Comment by twstagg** [comment]
> Duplicate of #1973
> 
> Or is it?

These are 2 separate issues I am intending to report. See either steps to reproduce. The series of steps to reproduce each issue are separate, as are the mechanisms I am reporting issues with.

**Comment by Yimura** [comment]
I'm guessing this is fixed with #2106, if you can confirm the bug still happens I'll reopen the issue.
